.github/workflows/desiger-check-quartz-update.yml

```
@@ -0,0 +1,64 @@
+    runs-on: ubuntu-latest
+    env:
+      tables-file: backend/src/Designer/Migrations/SqlScripts/QuartzTables/tables_postgres.sql
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Check if quartz library is updated
+        id: check-quartz-update
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)